docker-compose.yml

```diff
@@ -55,6 +75,13 @@ services:
         condition: service_healthy
       flink-taskmanager:
         condition: service_started
+      nessie:
+        condition: service_started
+    # For some reasons this dependency makes minio-setup stuck in Waiting state. Because of that only dependency to minio is used
+#      minio-setup:
```

Comment:
let's try to add `restart: "no"` to `minio-setup` service definition

Reply:
"no" policy is the default: https://docs.docker.com/engine/containers/start-containers-automatically/  but I can try